web/admin/src/SurveyResults.js

```diff
@@ -190,12 +190,15 @@ class SurveyResults extends Component {
   }
 
   prepareCsv = results => {
-    const attendeeClickPromises = results.map(result =>
-      this.props.client
-        .getAttendee(result.creator.id)
-        .then(attendee => ({ ...result, ...attendee }))
-        .catch(err => result),
-    )
+    const attendeeClickPromises = results.map(result => {
+      if (result.creator.id) {
+        return this.props.client
+          .getAttendee(result.creator.id)
+          .then(attendee => ({ ...result, ...attendee }))
+          .catch(err => result)
+      }
+      return result
```

Comment:
is `result` a Promise?

Reply:
It is not but this object needs to wait for the promises to finish. Is there a better way to structure this or would this cause issues?

Reply:
👍  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#Fulfillment